docs/migration.adoc

```diff
@@ -120,11 +149,39 @@ Then install the wrapper script to auto detect the version of rdiff-backup.
 
 ===== On CentOS
 
-TODO
+This installs the Python2 application `rdiff-backup-1` side-by-side with
+any other version `rdiff-backup` also installed.
+
+CentOS and RHEL 7 (From COPR)
+ 
+----
+sudo yum install yum-plugin-copr epel-release
+sudo yum copr enable frankcrawford/python2-rdiff-backup
+sudo yum install python2-rdiff-backup
+----
+ 
+CentOS, AlmaLinux, Rocky Linux and RHEL 8 (From COPR)
+ 
+----
+sudo dnf install dnf-plugins-core epel-release
+sudo dnf copr enable frankcrawford/python2-rdiff-backup
+sudo dnf copr enable frankcrawford/python2-pyxattr
+sudo dnf --enablerepo=PowerTools install python2-rdiff-backup
+----
+
+Fedora (From COPR)
+ 
+----
+sudo dnf install dnf-plugins-core epel-release
+sudo dnf copr enable frankcrawford/python2-rdiff-backup
+sudo dnf copr enable frankcrawford/python2-pyxattr
+sudo dnf copr enable frankcrawford/python2-pylibacl
+sudo dnf install python2-rdiff-backup
+----
```

Comment:
If I'm not mistaking, it's exactly the same as above, why not just [reference](https://docs.asciidoctor.org/asciidoc/latest/macros/xref/)?